.github/workflows/on-push-do-docs.yml

```diff
@@ -0,0 +1,22 @@
+name: on-push-do-docs
+on:
+  push:
+jobs:
+  docs:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run MarkdownSnippets
```

Comment:
Would be good to install the .NET SDK here so that it uses the same version as the code:

https://github.com/justeat/httpclient-interception/blob/46316fa2c0d74beb541c2086fdbeee714b339976/.github/workflows/build.yml#L33-L34

Reply:
to what value? AFAIK each workflow runs on a diff VM

Reply:
So that all the dotnet-related stuff running uses the SDK version in the global.json file, rather than whatever happens to be installed on the runner VMs on any given day.

Reply:
fair enough :) done